packages/server-wallet/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "module": "commonjs",
+    "target": "es2017",
+    "noImplicitAny": false,
+    "rootDir": "src",
+    "outDir": "lib",
+    "esModuleInterop": true,
+    "strict": false
```

Comment:
If we can avoid having to set this to `false` that would be nice.

Reply:
https://github.com/statechannels/monorepo/issues/2327